src/test/java/com/example/gooddeedsbe/JobValidatorTests.java

```diff
@@ -0,0 +1,255 @@
+package com.example.gooddeedsbe;
+
+import com.example.gooddeedsbe.exceptions.*;
+import com.example.gooddeedsbe.model.Job;
+import com.example.gooddeedsbe.utils.JobValidator;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class JobValidatorTests {
```

Comment:
Unit tests for specific class should reside in separate test class. For example JobValidator - JobValidatorTest, JobService - JobServiceTest.